src/components/NotificationsDropdown.tsx

```diff
@@ -217,14 +387,41 @@ export default function NotificationsDropdown(props: NotificationsDropdownProps)
     await populateNotifications();
   };
 
-  const markAsRead = async (read: boolean, id: number, close?: boolean) => {
-    if (close) {
-      onPopoverClose();
-    }
-    await MarkNotificationRead(read, id);
+  const markUserNotificationAsRead = async () => {
+    await PreferencesService.updateUserPreferences({
+      unitsAcknowledgedOnMs: Date.now(),
+      timeZoneAcknowledgedOnMs: Date.now(),
+    });
+
+    await reloadPreferences();
+    onPopoverClose();
```

Comment:
why are we closing here? we should move this to the top maybe so user is not waiting for the api to finish

Reply:
may need to check with Ming on where this notification should disappear or always be present